src/views/events/EventsDashboard.vue

```diff
@@ -118,8 +119,22 @@ function groupBy<T>(xs: T[], getKey: (element: T) => string) {
   }, {});
 }
 
+function findNextEventIndex() {
+  return actionsEvents?.value?.events.findIndex(
+    (event: Event) => new Date(event.end_time) > new Date(Date.now()),
+  );
+}
+
 const eventsWithFormattedDate = computed(() => {
-  const events = actionsEvents?.value?.events?.map((event: Event) => {
+  const urlParams = new URLSearchParams(window.location.search);
+  let earlier = 0;
+  if (urlParams.has("earlier")) {
+    earlier = Number(urlParams.get("earlier"));
+  }
+  const current_events = actionsEvents?.value?.events?.slice(
+    findNextEventIndex() - Math.max(0, earlier),
```

Comment:
Use `Math.max(0, findNextEventIndex() - earlier)` instead since you want to avoid passing a negative index to slice which is possible at the moment if `findNextEventIdex()` returns 3 and `earlier` is 5, for example.